src/buttons.css

```diff
@@ -190,31 +186,37 @@ against .round class */
 .btn.btn--pill-hr {
   border-top-left-radius: 0 !important;
   border-bottom-left-radius: 0 !important;
+  margin-left: -3px !important;
 }
 
 .btn.btn--pill-vc {
   border-radius: 0 !important;
   margin-top: 1px;
   margin-bottom: 1px;
+  display: block;
+  width: 100%;
 }
 
 .btn.btn--pill-vt {
   border-bottom-right-radius: 0 !important;
   border-bottom-left-radius: 0 !important;
+  display: block;
+  width: 100%;
 }
 
 .btn.btn--pill-vb {
   border-top-right-radius: 0 !important;
   border-top-left-radius: 0 !important;
+  display: block;
+  width: 100%;
 }
```

Comment:
Here we could safely use a fancy attribute selector, I think, if we really wanted to:

```css
.btn[class*='btn--pill-'] {
  display: block;
  width: 100%;
}
```